Phishing remediation enhamcements (#25151)
demisto/content: Packs/Phishing/ReleaseNotes/3_5_8.md

#### Layouts

##### Phishing Incident v3

> - Added "Selected indicators to block" section. this section will show the indicators chosen to be blocked during the investigation.

Reviewer: ```suggestion
Added "Selected indicators to block" section. This section will show the indicators that were chosen to be blocked during the investigation.
```

> - Added "Selected indicators to block" section and incident field. this section will show the indicators chosen to be blocked during the investigation.

Reviewer 2: Just elaborate on the new section ( since the field was removed).

#### Playbooks

##### Phishing - Generic v3

> - "Block Indicators - Generic v2" sub-playbook replaced with "Block Indicators - generic v3" sub-playbook.

Reviewer: ```suggestion
- The "Block Indicators - Generic v2" sub-playbook was replaced with the "Block Indicators - generic v3" sub-playbook.
```

> - "Block Indicators - Generic v2" sub-playbook replaced with "Block Indicators - generic v3" and the conditional task "Should indicators be blocked automatically?" has been removed. Block indicators method will be determined in the sub-playbook itself using the "AutoBlockIndicators" input.

Reviewer 2: - "Block Indicators - Generic v2" sub-playbook replaced with "Block Indicators - generic v3" sub-playbook.
- the conditional task "Should indicators be blocked automatically?" has been removed and handled by the new sybplaybook (handled by the `BlockIndicators` input).

> - the conditional task "Should indicators be blocked automatically?" has been removed and block indicators method(Manual/Automatic) being handled by the new sub-playbook (using the BlockIndicators input).

Reviewer: ```suggestion
- The conditional task "Should indicators be blocked automatically?" has been removed and block indicators method(Manual/Automatic) is being handled by the new sub-playbook (using the BlockIndicators input).
```

> - "BlockIndicators" input will now determine whether to automatically block indicators or selecting manually which indicators to block, if any.

Reviewer 2: Add the full description that you have edited for this input. will make sense to the users to review that once they updated the pack.

> - "BlockIndicators" input manages the automated block Indicators capability. Set to "True" for automatically block all malicious indicators, or set to "False" for choosing manually which indicators to block, if any.

Reviewer: ```suggestion
- "BlockIndicators" input manages the automated block indicators capability. Set to "True" to automatically block all malicious indicators, or set to "False" to manually choose which indicators to block, if any.
```

> - "Manually search & delete emails" manual task has been added and will be used when the user will choose to not search and delete emails automatically.

Reviewer 2: - "Manual Remediation" section and "Manually remediate the incident" task has been removed. Instead, the "Manually search & delete emails" manual task has been added and will be used when the user will choose to not search and delete emails automatically.

(combine the last 2 items into 1)

> - "Manual Remediation" section and "Manually remediate the incident" task has been removed. Instead, the "Manually search & delete emails" manual task has been added and will be used when the user will choose to not search and delete emails automatically.

Reviewer: ```suggestion
- The"Manual Remediation" section and "Manually remediate the incident" task have been removed. Instead, the "Manually search & delete emails" manual task has been added and will be used when the user chooses not to search and delete emails automatically.
```
